packages/bot-skeleton/src/scratch/help-content/help-content.config.js

```diff
@@ -7,7 +7,9 @@ export const help_content_types   = {
 
 const { TEXT, IMAGE, BLOCK }     = help_content_types;
 
-export const help_content_config = {
+export const help_content_config = (public_path) => {
+    __webpack_public_path__ = public_path; // eslint-disable-line no-global-assign
```

Comment:
Could we set the `__webpack_public_path__` globally for `bot-skeleton` and use that instead?

Reply:
`bot-web-ui` have a different `__webpack_public_path__` with `bot-skeleton`. Using `bot-skeleton` public path will result in 404 when retrieving image in help content.